README.md

```diff
@@ -29,7 +29,7 @@ To build Spark and its example programs, run:
 ./build/mvn -DskipTests clean package
 ```
 
-(You do not need to do this if you downloaded a pre-built package.)
+*You do not need to do this if you downloaded a pre-built package*.
```

Comment:
hm, not sure. either way doesn't much matter to me. I would just leave it as was.

Reply:
Let me disagree. Normally in the English language punctuation comes after the bracket. 
"Something like using other fonts (like Italic), or anything else is better than ("." at the end of the sentence.)."

Reply:
This does not matter either way, so would leave it, but, punctuation belongs inside the closing paren in a parenthesized sentence. I can't parse your example.